client/src/app/site/motions/services/motion-pdf.service.ts

```diff
@@ -103,14 +103,18 @@ export class MotionPdfService {
             crMode = this.meetingSettingsService.instant(`motions_recommendation_text_mode`);
         }
 
-        const title = this.createTitle(motion, crMode, lineLength);
-        const sequential =
-            infoToExport?.includes(`id`) ?? this.meetingSettingsService.instant(`motions_show_sequential_number`);
-        const subtitle = this.createSubtitle(motion, sequential);
+        if (!flowText) {
+            const title = this.createTitle(motion, crMode, lineLength);
+            const sequential =
+                infoToExport?.includes(`id`) ?? this.meetingSettingsService.instant(`motions_show_sequential_number`);
+            const subtitle = this.createSubtitle(motion, sequential);
 
-        motionPdfContent = [title, subtitle];
+            motionPdfContent = [title, subtitle];
+        } else {
+            motionPdfContent = [];
+        }
```

Comment:
The `else`-block is not necessary, since the `motionPdfContent` is initialized by `[ ]`.